cmd/zdb/zdb.c

```diff
@@ -2634,8 +2634,8 @@ print_indirect(spa_t *spa, blkptr_t *bp, const zbookmark_phys_t *zb,
 		}
 		if (BP_GET_LEVEL(bp) != zb->zb_level) {
 			(void) printf(" (ERROR: Block pointer level "
-			    "(%llu) does not match bookmark level (%ld))",
-			    BP_GET_LEVEL(bp), zb->zb_level);
+			    "(%llu) does not match bookmark level (%lld))",
+			    BP_GET_LEVEL(bp), (u_longlong_t)zb->zb_level);
```

Comment:
`zb_level` is an `int64_t`, this should cast this to a `longlong_t`.

Reply:
Fixed, thank you.